packages/arb-ts/package.json

```diff
@@ -19,7 +19,7 @@
     "postinstall": "$npm_execpath run gen:abi",
     "prepublishOnly": "yarn build && yarn format",
     "preversion": "yarn lint",
-    "build": "./scripts/builder",
+    "build": "./scripts/builder && cp -R ./src/lib/abi/*.d.ts ./dist/lib/abi",
```

Comment:
why is this necessary?

Reply:
Apparently tsc doesn't copy d.ts files, it only creates them. https://stackoverflow.com/questions/56018167/typescript-does-not-copy-d-ts-files-to-build

So I'm manually copying them as part of the build